apps/studio/src/features/editing-experience/components/form-builder/hooks/useS3Image.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react"
```

Comment:
for my understanding can i check when is useImage used over useS3Image? I believe all image uploads will now be using useS3Image right?

Reply:
`useImage` is used to handle uploading to s3 with the retries - this is required so that we don't run into the malware scan not being completed and get a 403.

`useS3Image` is when the image has already successfully uploaded and we want to download + show the image using the `FileAttachment` (as it requires a `file` but we only have a relative path `/<site_id>/<hash>/<image>`) 

i think i'll update `useImage` to `useImageUpload` for clarity in this case